db/seeds.rb

```diff
@@ -41,5 +53,43 @@
 
 # Create Aircrafts
 aircrafts.each do |aircraft_data|
-  Aircraft.create!(aircraft_data)
+  aircraft = Aircraft.create!(aircraft_data)
+  AIRCRAFT_NOTIFICATIONS_COUNT.times do
+    notification = NewAircraftCreatedNotification.with(aircraft: aircraft)
+    notification.deliver_later(NewAircraftCreatedNotification.targets)
+  end
+end
+
+# Create Logbook Entries and Notifications
+LOGBOOK_ENTRY_NOTIFICATIONS.times do
+  logbook_entry = LogbookEntry.create!(
+    aircraft: Aircraft.order("RANDOM()").first,
+    pilot_in_command: users.sample,
```

Comment:
seems that the `users` variable is not defined yet, running `rails db:seed` eventually gives me:

```
NameError: undefined local variable or method `users' for main (NameError)

    pilot_in_command: users.sample,
```

Reply:
@pJeyakumar , I just fixed this issue.